.gitlab-ci.yml

```diff
@@ -65,8 +67,9 @@ build-docker-image-cilium:
       CILIUM_ENVOY_IMAGE=registry.ddbuild.io/images/mirror/cilium/cilium-envoy:v1.29.7-39a2a56bbd5b3a591f69dbca51d3e30ef97e0e51@sha256:bd5ff8c66716080028f414ec1cb4f7dc66f40d2fb5a009fff187f4a9b90b566b
     TARGET: release
     NOSTRIP: 0
-  script:
-    - .gitlab/build-image.sh
```

Comment:
What are you referring to exactly? This particular change is just a YAML simplification because the `script` is already defined in the `build-docker-image` section above
https://github.com/DataDog/cilium/blob/2d078f7e7e37b62b46b002ac8961bf5552894506/.gitlab-ci.yml#L23-L24